datacommons/test/places_test.py

```diff
@@ -247,6 +247,60 @@ def test_multiple_dcids(self, urlopen):
             }
         })
 
```

Comment:
Have a test for validating multiple batches? (e.g., override _QUERY_BATCH_SIZE to a small value and expect >1 calls)